dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -33,14 +43,20 @@ export const CommercialMetrics: React.FC<{
 
 		if (isDev || shouldForceMetrics || userIsInSamplingGroup)
 			setSentCommercialMetrics(
-				sendCommercialMetrics(pageViewId, browserId, Boolean(isDev)),
+				sendCommercialMetrics(
+					pageViewId,
+					browserId,
```

Comment:
Another, non this PR related but perhaps useful, comment: @mxdvl it looks like we're not guarding against `browserId` being undefined here but we _could_, and maybe should?

We now know that if this prop is `undefined` then it means the cookie read hasn't completed and I think we want that to happen before sending these metrics. It's possible that this is always happening _by chance_ but it should be encoded, I think.

One simple, declarative, way to do this would be to just not render this component until `browserId` is defined using a conditional in `App.tsx`? That way we can make the prop required and the logic gets simplified

Reply:
(I'm pinging Max because we were talking about this earlier in another thread)

Reply:
It looks like roughly 10% of the rows in `fastly_commercial` have `browser_id is null`. 
I've added a TODO in the code.